WireAPI/Tests/WireAPITests/Helpers/APIServiceSnapshotHelper.swift

```diff
@@ -0,0 +1,152 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SnapshotTesting
+import enum WireAPI.APIVersion
+import XCTest
+
+@testable import WireAPI
+@testable import WireAPISupport
+
+/// A helper object to make snapshotting API requests easier.
+
+struct APIServiceSnapshotHelper<API> {
```

Comment:
This is kind of a duplication of the existing snapshot helper that only works with `HTTPClient`. I could try to remove the duplication, but I figured that we'll move away from `HTTPClient` and eventually discard it and related classes.

Reply:
can you create a ticket if there's some cleanup to do?

Reply:
It'll be part of the next milestone (retiring operation loop).